src/resolver.ts

```diff
@@ -62,7 +70,7 @@ export default {
         earlyData,
       );
 
-      yield pendingElements.every((element) => element !== undefined)
+      yield pendingElements?.every((element) => element !== undefined)
```

Comment:
```suggestion
      yield pendingElements!.every((element) => element !== undefined)
```

I don't know that this can actually be missing?